exercises/rust-1-a/example.rs

```diff
@@ -0,0 +1,35 @@
+// input: the hour, 1-12 inclusive
+// output: seconds after the hour at which the hands are best aligned
+pub fn hands_match(mut hour: u8) -> u32 {
+    hour %= 12;
+    // let's solve this analytically.
+    //
+    // other plausible solution strategies:
```

Comment:
Do you have any idea how plausible these other approaches are?

Reply:
I implemented the brute force version, which also took about 15 minutes. I didn't attempt binary search, but it seems plausible IMO.

Reply:
Good to hear!